### ISSUE TITLE
no current kernel found

### ISSUE THREAD

**Issue by havok2063** [mariobuikhuizen/voila-embed#22]
When testing the latest PR for 0.2.8 compatibility, https://github.com/mariobuikhuizen/voila-embed/pull/21, I'm now getting a `no current kernel` error message in the dev console.  

```
voila.js:455 Uncaught (in promise) Error: No current kernel
    at Ys._get_comm_info (voila.js:455)
    at Ys._build_models (voila.js:469)
    at voila-embed.js:165
``` 

This was discovered during testing of the new kernel pooling aspect (https://github.com/voila-dashboards/hotpot_km/pull/8) which is using the latest `voila==0.2.9` but this error also comes up when using the standard voila, version `0.2.7`.

**Comment by mariobuikhuizen** [comment]
Yes, #21 depends on the bug fix in this PR: https://github.com/voila-dashboards/voila/pull/876.

Maybe you can use this PR when testing?

**Comment by havok2063** [comment]
Ok let me try that.  What's weird is that yesterday everything was working properly but perhaps it shouldn't have been.

**Comment by havok2063** [comment]
So that PR is merged; I did a git pull and fresh install of `voila` from master, and I'm still getting the same error.

**Comment by mariobuikhuizen** [comment]
The file `voila-embed.js` is not part of the installation of voila-embed, it has to be copied manually to your project. Maybe this step is missing?

**Comment by havok2063** [comment]
Yeah I had copied the new file over into my project but still had the same problem.  I reset everything back to voila `0.2.7`, the old `embed`, closed everything down, refreshed, and got it back to a workable state.  I'll try again changing things piecemeal and see what happens.  Do you know what kind of thing would trigger this error, in case it happens again, I know what to think about or look for?

**Comment by mariobuikhuizen** [comment]
Sometimes modifying an environment doesn't work well, so I always test in a new conda environment. Also, the fix in Voila is released as 0.2.10, so you don't have to install Voila from git any more.

**Comment by havok2063** [comment]
Ok cool.  I'll upgrade everything, give it a shake, and see if everything works!

**Comment by havok2063** [comment]
Looks like everything is back to normal.  I've tested the `voila=0.2.10` and the latest `voila-embed` work in the standard mode as well as the `hotpot` kernel pooling mode.
